src/pyhf/cli/cli.py

```diff
@@ -0,0 +1,26 @@
+import logging
+
+import click
+
+from ..version import __version__
+from . import rootio, spec, stats
+
+logging.basicConfig()
+log = logging.getLogger(__name__)
+
+
+@click.group(context_settings=dict(help_option_names=['-h', '--help']))
+@click.version_option(version=__version__)
+def pyhf():
+    pass
+
+
+# pyhf.add_command(rootio.cli)
+pyhf.add_command(rootio.json2xml)
+pyhf.add_command(rootio.xml2json)
+
+# pyhf.add_command(spec.cli)
+pyhf.add_command(spec.inspect)
+
+# pyhf.add_command(stats.cli)
```

Comment:
These comments can get deleted now?
```suggestion
```

Reply:
depends on what we want to do with the api. If we want `pyhf rootio xml2json` and `pyhf rootio json2xml` etc...